src/Knoop/Node/test/assignment.test.cpp

```diff
@@ -0,0 +1,42 @@
+#include "catch.hpp"
+#include "Knoop.hpp"
+
+using namespace njoy::Knoop;
+using Node_t = Node<int, std::string>;
+
+SCENARIO( "assignment" ){
+  // this lambda is a workaround for isuues in the interaction between
+  // C macros (like Catch's REQUIRE statement) and C++ template
+  // instatiations;
+  auto getInt =
```

Comment:
Why don't we put these lambda functions in `Node.test.cpp` and make them available to all the other `cpp` files? Is that possible?

Reply:
To do so as a lambda would be difficult, but it would be pretty straight forward as a free function. Will change.

Reply:
Done!